modules/lib/lib-node/src/main/resources/lib/xp/node.js

```diff
@@ -294,7 +294,7 @@ RepoConnection.prototype.query = function (params) {
     handlerParams.setStart(params.start);
     handlerParams.setCount(params.count);
     handlerParams.setQuery(__.toScriptValue((params.query)));
-    handlerParams.setSort(valueOrDefault(params.sort, '_score DESC'));
+    handlerParams.setSort(__.toScriptValue(params.sort ? params.sort : '_score DESC'));
```

Comment:
1. before `params.sort = ''` would result in empty sort, now it would fall back to `_score DESC`
2. actually, I don't agree that if parameter is missing it should be `_score DESC` . It does not seem to be the same in content-lib - need to investigate